tests/plugins/org.polarsys.capella.test.model.ju/src/org/polarsys/capella/test/model/ju/menu/CapellaMenusTestCase.java

```diff
@@ -115,28 +119,31 @@ protected void testCopyAsTextCommand(EObject selectedElement) {
     }
   }
   
-  protected void testCopyAsTextNumericValue(Integer value) {
-    PrimitiveWrapper wrapper = new PrimitiveWrapper(value);
-    ExecutionEvent event = createExecutionEvent(wrapper);
+  protected void testCopyAsTextNumericValue(Integer... values) {
+    Object[] wrappers = Arrays.stream(values).map(value -> new PrimitiveWrapper(value)).toArray();
+    String valuesStr = Arrays.stream(values).map(value -> String.valueOf(value))
+        .collect(Collectors.joining(ICommonConstants.LINE_SEPARATOR));
```

Comment:
i was suggesting to do the join in the copy text handler :)

https://github.com/eclipse/capella/pull/2144/files#diff-c092a82eb8d039193021d884a74ccfdff4e50897d70bf2aa2dbc1b86a4f4c64eR62

Reply:
Done.